### ISSUE TITLE
[Crash]: 1.8.6 Client crash on joining server: "elementalcraft:pipe_cover_hiding used before it was bound"

### ISSUE THREAD

**Issue by TheInfamousAlk** [AllTheMods/ATM-6#2585]
### Possible Fixes

Yes

### Modpack Version

other

### What happened?

Upon upgrading to 1.8.6 on both server and client, I am getting the following client crash. I am the server owner. There are no custom mods, configs, or anything like that. The crash is not happening to anyone else on the server (4 other people). This is only happening upon upgrading from 1.8.5 to 1.8.6. 

Specifically, I get past "encrypting" and "joining world" and at that point it freezes up, then says "saving world" and hard crashes. To be clear, the server never crashes. Just my client. 

### Crash Log

https://gist.github.com/TheInfamousAlk/756675777c892cff67b66cf320a1ef00#file-crash-2021-10-15_10-04-03-client-txt

**Comment by TheInfamousAlk** [comment]
I completely wiped my modpack install and redownloaded it, same crash. I had a friend on the server go to where I was trying to log in and they do not crash, so it's not a world corruption issue as far as I can tell.

**Comment by TheInfamousAlk** [comment]
I have found a workaround. If I load up a single player world then leave back to menu, I can then join the server without a crash. This makes me think maybe it is related to the fact that in 1.8.5 and below, when I would try to join the server, it would fail and boot me back to the server list the first time I tried to join after loading the pack. It would always work the second time. So maybe this is the same thing, except instead of booting me to server list, it crashes the pack? I can't give any info on the previous error since it no longer happens. Maybe it's coincidence.
